src/main/java/com/acme/conferencesystem/ticket/http/TicketController.java

```diff
@@ -0,0 +1,54 @@
+package com.acme.conferencesystem.ticket.http;
+
+import com.acme.conferencesystem.ticket.business.Ticket;
+import com.acme.conferencesystem.ticket.business.TicketService;
+import jakarta.validation.Valid;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PatchMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+import java.util.UUID;
+
+@RestController
+@RequestMapping("/tickets")
+public class TicketController {
+
+    private final TicketService ticketService;
+
+    public TicketController(TicketService ticketService) {
+        this.ticketService = ticketService;
+    }
+
+    @GetMapping
+    public ResponseEntity<List<Ticket>> getAllTickets() {
+        return ResponseEntity.ok(ticketService.getAllTickets());
+    }
+
+    @PutMapping("/buy")
```

Comment:
I think it should be POST for creation, normally PUT is for full updates, and PATCH is for partial updates.